homeassistant/components/tuya/select.py

```diff
@@ -342,6 +342,31 @@
             translation_key="countdown",
         ),
     ),
+    # Dehumidifier
+    # https://developer.tuya.com/en/docs/iot/categorycs?id=Kaiuz1vcz4dha
+    "cs": (
+        SelectEntityDescription(
+            key=DPCode.COUNTDOWN_SET,
+            name="Countdown",
+            entity_category=EntityCategory.CONFIG,
+            icon="mdi:timer-cog-outline",
+            translation_key="countdown",
+        ),
+        SelectEntityDescription(
+            key=DPCode.FAN_SPEED_ENUM,
+            name="Fan Speed",
+            entity_category=EntityCategory.CONFIG,
+            icon="mdi:fan",
+            translation_key="fan_speed",
+        ),
```

Comment:
This should be a fan, not a select entity.

Reply:
Oh I oversee that entity, that's really better!